pkgs/top-level/all-packages.nix

```diff
@@ -2014,6 +2014,8 @@ with pkgs;
 
   sorted-grep = callPackage ../tools/text/sorted-grep { };
 
+  sozi = callPackage ../applications/office/sozi { };
+
```

Comment:
When moving the package into the `pkgs/by-name` tree this change is no longer necessary
```suggestion
```